src/test/java/shop/VirtualItemTest.java

```diff
@@ -0,0 +1,28 @@
+package shop;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class VirtualItemTest {
+    private VirtualItem virtualItem;
+
+    @BeforeEach
+    public void setUp() {
+        virtualItem = new VirtualItem();
+    }
+
+    @Test
+    void getWeight() {
+        final double expected = 1232.22;
```

Comment:
It was a trap. It doesn't make sense to create tests for getters/setters/etc. For RealItem/VirtualItem classes there are 2 correct solutions here:
1. Do not implement unit tests at all for them
2. Add test for toString method as far as it was overriden

Please update